tests_script.py

```diff
@@ -0,0 +1,4 @@
+from firefox_ui_harness import cli
+
+if __name__ == '__main__':
+    cli()
```

Comment:
I'm not that happy with those scripts especially in the root folder. So I would suggest to place them under `firefox_ui_harness/update.py` and respectively. Then we can move out the `cli` code from runtests.py, and update setup.py to make use of the new cli wrapper scripts.